include/extractor/external_memory_node.hpp

```diff
@@ -27,9 +37,12 @@ struct ExternalMemoryNode : QueryNode
 struct ExternalMemoryNodeSTXXLCompare
 {
     using value_type = ExternalMemoryNode;
-    bool operator()(const ExternalMemoryNode &left, const ExternalMemoryNode &right) const;
-    value_type max_value();
-    value_type min_value();
+    value_type max_value() { return value_type::max_value(); }
+    value_type min_value() { return value_type::min_value(); }
+    bool operator()(const value_type &left, const value_type &right) const
```

Comment:
If you mean definitions, then yes they have to be inline. But member function definitions inside the class body are already inline by default. This would only be a problem if we would do it like this:

    struct S { void fn(); };
    inline S::fn() { } // <- has to be inline, otherwise violates ODR